platform/backend/bridge/src/main/kotlin/io/hamal/bridge/web/code/GetCodeController.kt

```diff
@@ -0,0 +1,30 @@
+package io.hamal.bridge.web.endpoint.code
+
+import io.hamal.core.adapter.GetCodePort
+import io.hamal.lib.domain.vo.CodeId
+import io.hamal.lib.domain.vo.CodeVersion
+import io.hamal.lib.sdk.api.ApiCode
+import io.hamal.repository.api.Code
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+internal class CodeGetBridgeController(private val getCode: GetCodePort) {
+    @GetMapping("/b1/code/{codeId}")
+    fun getCode(
+        @PathVariable("codeId") codeId: CodeId,
+        @RequestParam(required = false, name = "codeVersion", defaultValue = "0") codeVersion: Int
```

Comment:
```suggestion
        @RequestParam(required = false, name = "code_version", defaultValue = "0") codeVersion: Int
```

In http land it is common to use snake_case instead of camelCase - therefore we follow this standard

Reply:
same for api controller - after this change some tests should fail if there are good

Reply:
Thought about that for a while - let just call it version as it clear in this context